### PR TITLE
fix: Removed inaccessible link & added user request info

### LINKED ISSUE #6818: Ticket Email: Ticket Download not working. Wrong URL
The ticket download email comes with a wrong URL to something like "https://api.eventyay.com/generated/tickets/orders/tickets/pdf/9bdedaa....".

It results in a page: "Not Found
The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again."

Screenshot from 2020-02-05 03-13-13
Screenshot from 2020-02-05 03-12-36

### PR DESCRIPTION
**Pull request by mrsaicharan1**
Fixes #6818

#### Short description of what this resolves:

- Removed inaccessible ticket link for security reasons

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Add a link to my tickets section

File: app/api/helpers/system_mails.py [hunk 1]
```diff
@@ -188,7 +188,8 @@
         'message': (
             u"Hi, this is a confirmation mail of your tickets for the event {event_name}"
             u"<br/>Your order has been processed successfully."
-            + u"<br/> <a href='{pdf_url}'>Click here</a> to view/download your ticket."
```
Comment (kushthedude) [inline app/api/helpers/system_mails.py]: Can we also remove the pdf_url attribute from the api/mail.py in `send mail attendee function`?

Comment (mrsaicharan1) [reply app/api/helpers/system_mails.py]: I'm not sure I understand. Could you point me to the permalink?

File: app/api/helpers/mail.py [hunk 2]
```diff
@@ -351,7 +351,8 @@ def send_email_to_attendees(order, purchaser_id, attachments=None):
                     event_name=order.event.name, invoice_id=order.invoice_number
                 ),
                 html=MAILS[TICKET_PURCHASED_ATTENDEE]['message'].format(
-                    pdf_url=holder.pdf_url, event_name=order.event.name
+                    mytickets_url=get_settings()['frontend_url'] + '/my-tickets',
```
Comment (iamareebjamal) [inline app/api/helpers/mail.py]:
```suggestion
                    my_tickets_url=get_settings()['frontend_url'] + '/my-tickets',
```

File: app/api/helpers/system_mails.py (hunk 1 above)
Comment (kushthedude) [reply app/api/helpers/system_mails.py]:
```py
                html=MAILS[TICKET_PURCHASED]['message'].format(
                    pdf_url=holder.pdf_url,
                    event_name=order.event.name,
                    frontend_url=get_settings()['frontend_url'],
                ),
```
pdf_url here can be removed after this change .

**Comment by mrsaicharan1** [comment]
travis seems to be failing because of this.
`__main__:1: DeprecationWarning: The import 'werkzeug.secure_filename' is deprecated and will be removed in Werkzeug 1.0. Use 'from werkzeug.utils import secure_filename' instead`

**Comment by iamareebjamal** [comment]
That's a warning, not an error

**Comment by mrsaicharan1** [comment]
Oops. Meant by this. `Failure: ImportError (cannot import name 'secure_filename' from 'werkzeug'`

**Pull request merged**